reference_algorithms/target_setting_algorithms/README.md

```diff
@@ -17,8 +17,8 @@ python3 submission_runner.py \
     --experiment_dir=$ROOT_DIR \
     --experiment_name=target_setting \
     --workload=criteo1tb \
-    --submission_path=reference_algorithms/target_setting_runs/jax_adamw.py \
-    --tuning_search_space=reference_algorithms/target_setting_runs/criteo1tb/tuning_search_space.json
+    --submission_path=reference_algorithms/reference_algorithms/target_setting_algorithms/jax_adamw.py \
+    --tuning_search_space=reference_algorithms/reference_algorithms.target_setting_algorithms/criteo1tb/tuning_search_space.json
```

Comment:
I think there is a `.` where there should be a `/`.

Reply:
Ah good catch thanks